packages/backend/src/services/pets/pets.model.ts

```diff
@@ -0,0 +1,15 @@
+import NeDB from 'nedb';
+import path from 'path';
+import { Application } from '../../declarations';
+
+export default (app: Application): NeDB<any> => {
+  const dbPath = app.get('nedb');
+  const Model = new NeDB({
+    filename: path.join(dbPath, 'pets.db'),
+    autoload: true,
```

Comment:
Please add the `inMemory` line from `users.model.ts` here as well.

Reply:
fixed in c9562eec904cabada1f650c950ed9ae88aacd317